ext/pdo/pdo_stmt.stub.php

```diff
@@ -5,13 +5,13 @@
 class PDOStatement implements IteratorAggregate
 {
     /** @return bool */
-    public function bindColumn(string|int $column, mixed &$param, int $type = 0, int $maxlen = 0, mixed $driverdata = null) {}
+    public function bindColumn(string|int $column, mixed &$param, int $type = 0, int $max_length = 0, mixed $driver_options = null) {}
 
     /** @return bool */
-    public function bindParam(string|int $parameter, mixed &$param, int $type = PDO::PARAM_STR, int $maxlen = 0, mixed $driverdata = null) {}
+    public function bindParam(string|int $param, mixed &$bind_var, int $type = PDO::PARAM_STR, int $max_length = 0, mixed $driver_options = null) {}
```

Comment:
I could also imagine `$var` instead of `$bind_var`

Reply:
I could be convinced of that if others feel strongly.  I felt "var" was too generic, and when writing it in named form `bind_var` seemed more descriptive.